test/client/main.cpp

```diff
@@ -281,9 +263,9 @@ int main(int argc, char* argv[])
     bool ensure_blob_class = false;
     bool download_first = false;
     // Client reset options are also used for async open.
-    std::string client_reset_metadata_dir = "";
-    bool disable_client_reset_recover_local_changes = false;
-    bool disable_client_reset_require_recent_state_realm = false;
+    // std::string client_reset_metadata_dir = "";
```

Comment:
Can we remove these if they aren't supported anymore?

Reply:
Removed